Resources/Prototypes/Entities/Mobs/NPCs/nymph.yml

```diff
@@ -0,0 +1,99 @@
+- type: entity
+  name: Nymph
```

Comment:
nitpick but please structure your yaml according to the [conventions](https://docs.spacestation14.io/en/getting-started/conventions).
![image](https://user-images.githubusercontent.com/98561806/210269869-93fed4de-f207-4534-8abb-c91a3bf5f41a.png)